NitroxPatcher/Patches/Persistent/UnityEngine_SetupCoroutine_Patch.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Reflection;
+using UnityEngine;
+
+namespace NitroxPatcher.Patches.Persistent;
+
+/// <summary>
+/// Makes sure that Coroutine aren't fully blocked once they encounter an Exception
+/// </summary>
+public sealed partial class UnityEngine_SetupCoroutine_Patch : NitroxPatch, IPersistentPatch
+{
+    public static readonly MethodInfo TARGET_METHOD = Assembly.GetAssembly(typeof(GameObject))
+                                                              .GetType("UnityEngine.SetupCoroutine")
+                                                              .GetMethod("InvokeMoveNext", BindingFlags.Public | BindingFlags.Static);
+
+    public static Exception Finalizer(Exception __exception)
+    {
+        if (__exception == null)
+        {
+            return null;
+        }
+        // TODO: Remove before merging
+        Log.Info($"Finalizer skipped an exception: {__exception}");
+        return null;
```

Comment:
I expect we would want to see errors it debug mode as I expect it can give us insight. Though for release/users it's most likely noise.